### ISSUE TITLE
500 Internal Server Errors Returned by public-rest32.*

### ISSUE THREAD

**Issue by VR51** [bullhorn/career-portal#136]
We are using the Bullhorn Career Portal. We have an Access Token and Swimlane but the message returned by your server is a 500 Internal Server Error.

https://public-rest32.bullhornstaffing.com/rest-services/213S9D/query/JobBoardPost?where=(isOpen=true)&fields=id,title,publishedCategory(id,name),address(city,state),employmentType,dateLastPublished,publicDescription,isOpen,isPublic,isDeleted&count=20&orderBy=-dateLastPublished&start=0

Can you suggest a solution please. The portal is at jobs.aicila.com.

**Comment by bulbajackel** [comment]
Hello,

To finalize the setup, you will want to contact Bullhorn Support.  They will be able to complete the setup once notified.  To contact Bullhorn Support you will want to follow the steps in the article below.

http://bullhorn.force.com/pkb/articles/Documentation/Contacting-Bullhorn-Support

Please let me know if you have any questions

**Comment by VR51** [comment]
Thanks Bulbajackel.

Can I make a suggestion for the readme.md file?

When I read the installation instructions in the readme.md I took away from them that I must build the Career Portal then upload it. Wasn't until I re-read the instructions after several visits that I realised the pre-built package is available for download and ready installation; and I only picked up on this when I spotted the built from source package was a similar size to the downloadable package. You might want to reword the instructions in the readme.md to make it clear that the package can be built from source (575kb download) or downloaded as a prebuilt package ready for deployment and configuration (5.75MB package).

There are many dependency updates (security and bug fixes) that need to be addressed in the source code package.json and bower.json files too.

Thanks again!
